Solutions/Tanium/Workbooks/TaniumWorkbook.json

```diff
@@ -0,0 +1,792 @@
+              },
+              "name": "query - 3"
+            },
+            {
+              "type": 1,
+              "content": {
+                "json": "## THR Alert by User\n"
+              },
+              "name": "text - 4"
+            },
+            {
+              "type": 3,
+              "content": {
+                "version": "KqlItem/1.0",
+                "query": "TaniumThreatResponse_CL\n| summarize count() by Match_Details_match_properties_user_s\n| render piechart  ",
+                "size": 0,
+                "timeContext": {
+                  "durationMs": 2592000000
+                },
+                "queryType": 0,
+                "resourceType": "microsoft.operationalinsights/workspaces"
+              },
+              "name": "query - 5"
+            },
+            {
+              "type": 1,
+              "content": {
+                "json": "## Threat Response Alerts by Process Name"
+              },
+              "name": "text - 6"
+            },
+            {
+              "type": 3,
+              "content": {
+                "version": "KqlItem/1.0",
+                "query": "TaniumThreatResponse_CL\n| where isnotempty(Match_Details_match_properties_name_s)\n| extend ActingProcessName = Match_Details_match_properties_name_s\n| summarize event_count=count() by bin(Timestamp_t, 7d), ActingProcessName\n| top 20 by event_count\n| render barchart",
+                "size": 0,
+                "timeContext": {
+                  "durationMs": 2592000000
+                },
+                "queryType": 0,
+                "resourceType": "microsoft.operationalinsights/workspaces"
+              },
+              "name": "query - 7"
+            },
+            {
+              "type": 1,
+              "content": {
+                "json": "## All Alerts by Computer Name"
+              },
+              "name": "text - 8"
+            },
+            {
+              "type": 3,
+              "content": {
+                "version": "KqlItem/1.0",
+                "query": "TaniumThreatResponse_CL\n|summarize count() by Computer_Name_s, Intel_Name_s\n|project-rename Computer_Name=Computer_Name_s, Intel_Name=Intel_Name_s, Count=count_\n|sort by Count",
+                "size": 0,
+                "timeContext": {
+                  "durationMs": 2592000000
+                },
+                "queryType": 0,
+                "resourceType": "microsoft.operationalinsights/workspaces"
+              },
+              "name": "query - 9"
+            }
+          ]
+        },
+        "conditionalVisibility": {
+          "parameterName": "selectedTab",
+          "comparison": "isEqualTo",
+          "value": "Threat Response"
+        },
+        "name": "Threat Response"
+      }
+    ],
+    "fallbackResourceIds": [
+      "/subscriptions/8cacf840-893d-4966-8f82-768f12f19076/resourcegroups/rg_1/providers/microsoft.operationalinsights/workspaces/log-sentinel-test"
```

Comment:
@ian-hands-tanium, A workbook template must not contain any references to resources, please update, thanks.

Reply:
@v-sabiraj the "FallbackResourceIDs" has been removed. Please let me know if there is anything else needed, thank you!

Reply:
@ajcollins0, its fine to add with empty array